BetaCameras/Sick.VisionaryT/VisionaryT.cs

```diff
@@ -79,9 +79,9 @@ private ListParamDesc<VisionaryTIntegrationTime> IntegrationTimeDesc
             }
         }
         /// <summary>
-        /// Integration time of ToF-sensor [ms].
+        /// Integration time of ToF-sensor [us].
         /// </summary>
-        public VisionaryTIntegrationTime IntegrationTime
+        public int IntegrationTime
         {
             get => _control.GetIntegrationTime();
             set => _control.SetIntegrationTime(value);
```

Comment:
If you use the Param-mechanism it will be checked automatically, because it's a RangeParam.
If you use the property directly there's no check.

This is consistent with the behavior of MetriCam2

Reply:
Any idea what would happen if a developer uses the property and sets an invalid value? Would it just clamp to the maxima/minima or cause some undefined behavior?
I don't think `consistent with the rest of the codebase` is a good argument in this case.
If by setting an invalid value the code can eat your hamster or cause problems with the hardware there should be checks.

Reply:
`SetIntegrationTime` uses `WriteVariable` which in turn checks the driver's response code. If setting an invalid value could actually harm the camera, then the camera should check that.

Reply:
So, what's the conclusion? Should I change something, or will you merge this PR?

Reply:
I already gave my OK for the PR. I would add checks here, but imo it's no blocker not to merge this.